module4/lessons/Week3/MaintainingState.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: page
+title: Maintaining State
+---
+
+## Learning Goals
+* Understand how cookies impact our experience on the internet
+* Implement a simple cookie
+* Recognize some limitations and regulations with cookies
+
+## Warm Up
+
+<section class='call-to-action' markdown='1'>
+
+**With a Partner**  
+Based on your experience interacting with websites:
+* What is a cookie?
+* How do you know if a website uses cookies?
+
+</section>
+
+
+## If you Give a Dev a Cookie
+
+HTTP is a stateless protocol.  This means that each request to a server exists in isolation - the server makes no connection between one request and the next. This makes for a very boring internet!  If we stayed in a stateless environment, we wouldn't be able to log in to applications, or save personal preferences, and the online shopping experience would be much more difficult.
+
+<section class='call-to-action' markdown='1'>
+
+**Independent Research**
```

Comment:
Love that we're moving to more of the responsibility on students! Also LOVE that there is an instructor note with the key takeaways you want to make sure come up in the conversation.